tests/test_queuing.py

```diff
@@ -122,21 +149,23 @@ def test_arbitrary_obj(self):
                 assert q.queue == objs
 
     def test_remove_not_in_queue(self):
-        """Test that remove does not fail when the object to be removed is not in the queue."""
+        """Test that remove fails when the object to be removed is not in the queue."""
```

Comment:
hmm, how come this behaviour changed?

Reply:
Ah, this was a change we decided to make in the queuing refactor ADR :)

The queuing refactor was never completed in the old core - it was partially implemented, and then completed fully in tape mode. So this is how tape queuing has worked since inception, and this is just updating the `test_queuing.py` file to align with the new queuing behaviour (as opposed to the partially complete queuing behaviour).